index.html

```diff
@@ -1 +1 @@
-<!DOCTYPE html><html><head><meta http-equiv="refresh" content="0; url=book/"></head><body></body></html>
+<!DOCTYPE html><html><head><meta http-equiv="refresh" content="0; url=book/overview/introduction.html"></head><body></body></html>
```

Comment:
cc @MyonKeminta See this line.

Reply:
Ouch, sorry i didn't notice that.

Reply:
Don't be! :) Very happy to have you look at my work.